examples/CREATE_AND_USE_A_SESSION.md

```diff
@@ -0,0 +1,86 @@
+### Create and Use a Session
+
+| Key                                                                                         | Description           |
+| ------------------------------------------------------------------------------------------- | --------------------- |
+| [sessionConfigs](https://bcnmy.github.io/biconomy-client-sdk/types/BuildUserOpOptions.html) | CreateSessionConfig[] |
+
+```typescript
+import {
+  createSmartAccountClient,
+  createSession,
+  createSessionSmartAccountClient,
+} from "@biconomy/account";
+import { createWalletClient, http, createPublicClient } from "viem";
+import { privateKeyToAccount, generatePrivateKey } from "viem/accounts";
+import { mainnet as chain } from "viem/chains";
+
+const account = privateKeyToAccount(generatePrivateKey());
+const signer = createWalletClient({ account, chain, transport: http() });
+const smartAccount = await createSmartAccountClient({
+  signer,
+  bundlerUrl,
+  paymasterUrl,
+}); // Retrieve bundler and pymaster urls from dashboard
+const smartAccountAddress = await smartAccount.getAccountAddress();
+
+const { sessionKeyAddress, sessionStorageClient } =
+  await createAndStoreNewSessionKey(smartAccount, chain);
```

Comment:
this will just create a new key pair? store it where?

Reply:
`createAndStoreNewSessionKey` has three arguments. The third is optional and isn't used here: `ISessionStorage`. If the third argument isn't used then it will pick a storage client for you by default: localStorage if in the browser and FileStorage if not

Reply:
current majorly used devx i have seen from dapps and demos is
i. client creates session keypair and passes session key eoa to us. keeps pk with them but pases ethers signer object

ii. during the op you pass which session key eoa and which SVM address (or session id) useful to find a leaf and corresponding proof - for dummy sig, original sig stuff

Reply:
This makes sense actually. Passing the whole store back to the sdk doesn't make sense. I thought it would be more convenient but it's less secure. I will simplify this a bit and make the store less engrained.